CHANGES.txt

```diff
@@ -1,3 +1,10 @@
+v0.4.1, 03 December 2018
```

Comment:
This should be `v0.4.0`

Reply:
I already released `0.4.0` on PyPi. then I amended my commit, which was a mistake. That's why you don't see `0.4.0` and `0.4.1` in the changelog. I'd just leave this be if you don't mind. But if you really think it matters, I can delete from PyPi and we can re-use this version.

Reply:
Hmmm. You pushed to PyPI from your repo and not PMA2020's (i.e. before the changes were brought into PMA2020's)? That might cause issues in the future, and there is a minor one now. I know you want less barriers in your development, but it would be more prudent to wait until changes are brought into PMA2020's repo.

Reply:
I hear you. I think the best option is to delete what I've got on PyPi for now, since I'm basically the only one importing it. I've done that. So I'm going to change the version to `0.4.0` now.